saleor/graphql/payment/mutations/base.py

```diff
@@ -74,6 +77,18 @@ def clean_source_object(
                     )
                 }
             )
+
+        if source_object.payment_transactions.count() >= cls.TRANSACTION_ITEMS_LIMIT:
+            raise ValidationError(
+                {
+                    "transactions": ValidationError(
```

Comment:
AFAIK the `field` should be the one used in the `input`. Based on the `field` value, storefront can identify incorrect field. You don't have `transactions` as an input. For `transactionInitialize` there is `id` field. It is checkout or order id. It sounds more reasonable to return the error with `id`, as the input ID is the one which caused the problem.